app/Renderers/Layouts/Campaign/CampaignExport.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace App\Renderers\Layouts\Campaign;
+
+use App\Renderers\Layouts\Layout;
+use Illuminate\Support\Facades\Storage;
+
+class CampaignExport extends Layout
+{
+    /**
+     * Available columns
+     * @return array[]
+     */
+    public function columns(): array
+    {
+        $columns = [
+            'created_by' => [
+                'key' => 'user.name',
+                'label' => 'campaigns.exports.created_by',
+                'render' => function ($model) {
+                    $html = '<a class="block break-all truncate" href="' . route('users.profile', [$model->user]) . '" target="_blank">' . $model->user->name . '</a>';
```

Comment:
Crashes if the user who created the export deletes their account, or if the export was triggered by a command line `sail artisan campaigns:export <id>` ;-)